code/modules/hazmat/crystal_processor.dm

```diff
@@ -0,0 +1,55 @@
+#define CRYSTAL_MACHINE_COLOR "colorizer"
+#define CRYSTAL_MACHINE_SHAPE "shaper"
+#define CRYSTAL_MACHINE_SIZE "sizer"
+/obj/machinery/hazmat/crystal_processor
+	name = "crystal processor"
+	desc = "Modify whatever crystal you got."
+	icon_state = "crystal_maker"
+	var/machinery_type = CRYSTAL_MACHINE_COLOR
+	var/processing_speech = "Doing the Needful to CRYSTAL"
+
+/obj/machinery/hazmat/crystal_maker/attackby(obj/item/I, mob/user, params)
+	. = ..()
+	if(istype(I, /obj/item/crystal))
```

Comment:
can you please start using early returns? this is something that has to be posted on every one of your PRs now.

Reply:
you are not a maintainer

Reply:
it's in the guidelines goof, and I might not be a maintainer but I am definitely free to review your PRs and give you feedback.

Reply:
but if it makes you happy I can ping a maintainer and they will tell you the exact same. @ninjanomnom

Reply:
show me the bylaws

Reply:
![image](https://user-images.githubusercontent.com/20987591/76756585-43438b00-6786-11ea-8d8a-c841f54a33c0.png)

Reply:
https://github.com/tgstation/tgstation/blob/master/.github/CONTRIBUTING.md#use-early-return

Reply:
fuck

Reply:
hm, maybe the non-maintainers who comment on your PRs are actually trying to help you/help improve your PRs, can actually have good advice/suggestions, and shouldn't be ignored/treated like human trash

but what do I know? I'm just a non-maintainer, after all